src/inventory.rs

```diff
@@ -319,8 +303,8 @@ impl DeploymentInventoryService {
             temp_dir_path.join(inventory.name.clone())
         };
 
-        let count = if inventory.peers.len() < DEFAULT_CONTACTS_COUNT {
-            inventory.peers.len()
+        let count = if inventory.peers().len() < DEFAULT_CONTACTS_COUNT {
+            inventory.peers().len()
```

Comment:
I couldn't highlight the code below this one, but we us the `count` to get random peers of len `count`, but the below code is prone to repetition of the peers. I think we can just use `choose_multiple(count)` there, but we might have to use the `rand` dependency.

Reply:
Sorry, I'm not sure I completely understand what the concern is here, but for this one also, there wouldn't really be anything different about how it's being done now vs. the previous implementation, so I don't think it's too concerning for the current PR.

Reply:
Yeah, it is not necesarily related to this PR. We can modify that later.

Reply:
I think on this one I will also leave it because I'm not sure exactly what change would be required.